docs/integrations/sources/postgres.md

```diff
@@ -232,12 +232,14 @@ SELECT pg_create_logical_replication_slot('airbyte_slot', 'wal2json');
 
 For each table you want to replicate with CDC, add the replication identity (the method of distinguishing between rows) first:
 
-To use primary keys to distinguish between rows, run:
-
+For tables that do not contain a large amount of data in a row, to use primary keys to distinguish between rows, run:
```

Comment:
Rephrase as: "To use primary keys to distinguish between rows for tables that don't have a large amount of data per row, run:"

Also, what's our guidance for tables that do contain large amounts of data?

Reply:
@Amruta-Ranade here is guidance for tables that do contain large amounts of data https://github.com/airbytehq/airbyte/pull/18291/files#diff-99b4439a040e2c24ad05b22b27a13bf9db80e0a42641be3d7d79e5a3dff4049fR239

Reply:
oh, duh, sorry about that!